sdk/core/azure-core/azure/core/pipeline/policies/_authentication.py

```diff
@@ -21,6 +22,47 @@
     from azure.core.pipeline import PipelineRequest, PipelineResponse
 
 
+class _HttpChallenge(object):  # pylint:disable=too-few-public-methods
```

Comment:
HttpChallenge?

Reply:
As in, remove the underscore? I can do that -- wasn't sure if we should since this class isn't exposed publicly, but it's imported in `_authentication_async.py` as well so that sounds reasonable

Reply:
It is in_authentication.py (has underscore) and used in another file.

Removing the underscore from the class name may be more reasonable :)